packages/block-editor/src/components/rich-text/index.js

```diff
@@ -128,6 +128,22 @@ function RichTextWrapper(
 		__unstableEmbedURLOnPaste,
 		__unstableDisableFormats: disableFormats,
 		disableLineBreaks,
+		unstableOnFocus,
+		__unstableAllowPrefixTransformations,
+		__unstableMultilineRootTag,
+		// Native props.
+		__unstableMobileNoFocusOnMount,
+		deleteEnter,
+		placeholderTextColor,
+		textAlign,
+		selectionColor,
+		tagsToEliminate,
+		rootTagsToEliminate,
+		disableEditingMenu,
+		fontSize,
+		fontFamily,
+		fontWeight,
+		fontStyle,
 		...props
```

Comment:
Can you clarify the changes to this component more? I mean why the props need to end up on different components now?

Reply:
Because we may be spreading more props onto RichText now. Previously we cherry picked the props that should be passed down to the editable element such as style and classes. Now any prop that is not RichText's props will be passed down.